### DISCUSSION TITLE
The application which scans thousands of repositories in an organisation and fetches .NET version, latest contributors and other info

### DISCUSSION THREAD

**Discussion by bonyaroslav** [octokit/octokit.net#2645] [Show and tell]
Hi
My organisation had a problem: there was no visibility of projects using the outdated Microsoft .NET framework.
There were thousands of them, and getting the whole list was impossible. Also, the issue was that ownership of them was questionable.
The resulting output of my new utility is CSV/excel file which contains not only project files, solutions names but
few latest contributors, GitHub teams and links to open for manual validation.

Would it be helpful to share such a unique application somewhere? I believe it may be useful for organisations to create such a report.
What are your thoughts?

**Comment by kfcampbell** [comment]
Hi @bonyaroslav! I think it'd be cool to share the source code or README file on this discussion if you'd like! That does sound useful. 

I would feel a little bit uncomfortable with you sharing direct links to a binary for it.
